py_neuromodulation/nm_database.py

```diff
@@ -0,0 +1,82 @@
+import os
+import sqlite3
+import time
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import glob
+
+class NMDatabase:
+    def __init__(self, out_path_root, folder_name):
+        self.out_path_root = out_path_root
+        self.folder_name = folder_name
+        self.db_time_idx = int(time.time() * 1000)
+        self.db_path = Path(out_path_root, folder_name, f"stream{self.db_time_idx}.db")
+        self.csv_path = Path(out_path_root, folder_name, f"stream.csv")
+        
+        pattern = str(Path(out_path_root, folder_name, "stream*.db"))
+
+        for file_path in glob.glob(pattern):
+            if os.path.exists(file_path):
+                if (self.db_time_idx/1000) - os.path.getctime(file_path) > 600:
+                    os.remove(file_path)
+
+        self.conn = sqlite3.connect(self.db_path, isolation_level=None)
+        self.cursor = self.conn.cursor()
+
+        db_dir = Path(folder_name)
+        if os.path.exists(db_dir):
```

Comment:
Check if necessary

Reply:
Indeed wasn't necessary in the latest version. Thanks!